### PR TITLE
{Network} Fix test_private_endpoint_connection_acfr, test_private_link_resource_acfr

### PR DESCRIPTION
**Pull request by jsntcy**
**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Review by evelyn-ys — APPROVED** [review]

**Comment by yonzhan** [comment]
Network

**Review by yonzhan — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_private_endpoint_connection_acfr.yaml [hunk 1]
```diff
@@ -37,8 +37,8 @@ interactions:
         {},\r\n    \"redisConfiguration\": {\r\n      \"maxclients\": \"1000\",\r\n
         \     \"maxmemory-reserved\": \"50\",\r\n      \"maxfragmentationmemory-reserved\":
         \"50\",\r\n      \"maxmemory-delta\": \"50\"\r\n    },\r\n    \"accessKeys\":
-        {\r\n      \"primaryKey\": \"xY3tVWeu5LlRK6sZrn7d02gX6kq9ZQhIRAzCaM0nhrI=\",\r\n
-        \     \"secondaryKey\": \"0hY15HHIXMKHUozFgVwnVTl8KPXELzE6zAzCaKQaLg8=\"\r\n
+        {\r\n      \"primaryKey\": \"abc\",\r\n
+        \     \"secondaryKey\": \"abc\"\r\n
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_private_endpoint_connection_acfr.yaml:41]: If YAML is recorded again, Credential Scan will fail again.

Comment (jsntcy) [reply src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_private_endpoint_connection_acfr.yaml]: Better way is to use replacer to replace credentials. Just removing credential to quick fix CI failure.

Comment (jiasli) [reply src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_private_endpoint_connection_acfr.yaml]: It was previously manually replaced. That's why we hit this pitfall again: https://github.com/Azure/azure-cli/pull/19967#discussion_r776609439.

**Review by jiasli — APPROVED** [review]

**Pull request merged**